lib/internal/test_runner/coverage.js

```diff
@@ -30,6 +30,7 @@ const { tmpdir } = require('os');
 const { join, resolve, relative, matchesGlob } = require('path');
 const { fileURLToPath } = require('internal/url');
 const { kMappings, SourceMap } = require('internal/source_map/source_map');
+const { getOptionValue } = require('internal/options');
```

Comment:
We've just finished a lot of refactoring to avoid using `getOptionValue()` throughout the test runner. Please don't reintroduce it.